test/unittests/parser_test.cpp

```diff
@@ -567,39 +567,40 @@ TEST(parser, milestone1)
     EXPECT_EQ(c.instructions,
         (std::vector{Instr::local_get, Instr::local_get, Instr::i32_add, Instr::local_get,
             Instr::i32_add, Instr::local_tee, Instr::local_get, Instr::i32_add, Instr::end}));
-    EXPECT_EQ(c.immediates, from_hex("00000000"
-                                     "01000000"
-                                     "02000000"
-                                     "02000000"
-                                     "00000000"));
+    EXPECT_EQ(c.immediates,
```

Comment:
In `block_br` you prefer the other way around:
```
EXPECT_EQ(hex(c.immediates), "00000000"...);
```

Reply:
The one in `block_br` is easier to debug because you see hex in the test failure output. But is probably less readable in the source code. I can convert these back to the compact form.